general.css

```diff
@@ -130,3 +129,29 @@ footer {
   list-style-type: none;
   gap: 10px;
 }
+
+@media screen and (max-width: 550px) {
+  body{
+    font-size: 1.2rem;
+  }
+  main{
+    padding:15px;
+  }
+  .footer-container{
+    justify-content:flex-start;
```

Comment:
Apart from the code formatting here in the CSS file, everything else looks good. The UI looks very nice @kiranpratyush 👍🏼